src/main/resources/i18n/strings.properties

```diff
@@ -336,6 +336,15 @@ preferences.contribute.registeredFor=Supporter certificate registered for %s
 preferences.contribute.noCertificate=Support Cryptomator and receive a supporter certificate. It's like a license key but for awesome people using free software. ;-)
 preferences.contribute.getCertificate=Don't have one already? Learn how you can obtain it.
 preferences.contribute.promptText=Paste supporter certificate code here
+preferences.contribute.thankYou=Thank you for supporting Cryptomator’s open-source development!
+preferences.contribute.donate=Donate
+preferences.contribute.sponsor=Sponsor
+
+### Remove License Key Dialog
+removeCert.title=Remove Cert
+removeCert.message=Remove supporter certificate?
+removeCert.description=Even without a supporter certificate Cryptomator stays functional and neither restrict access to your vaults nor lowers its security.
+removeCert.confirmBtn=Remove Cert
```

Comment:
Don't use the short version "Cert" here. For translators this is hard to translate. Either use the regular word (Certificate) or only use the verb "Remove".